config/chime_fpga_patterntest_recv.yaml

```diff
@@ -12,16 +11,18 @@
 type: config
 log_level: info
 num_elements: 2048
-num_local_freq: 1
-udp_packet_size: 4928
-num_data_sets: 1
-samples_per_data_set: 32768
 buffer_depth: 64
-num_gpu_frames: 128
-block_size: 32
-cpu_affinity: [4,5,6,7,12,13,14,15]
 num_ev: 4
 
+# CPU affinity for most processes
+#
+# Only core 7 is unused by the high priority processes. /stacking and
+# /apply_gains each have three threads and three dedicated cores (1, 2, 3 for
+# stacking, 4, 5, 6 for apply_gains), however as each of these needs only about
+# 2.5 cores of conpute power, we have opened up one of the hyperthread pairs to
+# other stages (11 and 14, twinned with 3 and 6 respectively)
```

Comment:
This comment seems to just be copied over from another file, and completely irrelevant.